### ISSUE TITLE
how can I ignore timeout?

### ISSUE THREAD

**Issue by 687766616e** [derv82/wifite2#216]

**Comment by RaduNico** [comment]
You will have to be more explicit than that, wifite has plenty of timeouts

**Comment by 687766616e** [comment]
all of them? (needed

**Comment by 0xCyberFortress** [comment]
I agree with this. a -ignore-timeout tag would be helpful. I'd like WPA/WPS timeouts to not exist, or to be able to change them to a longer setting, as I sometimes feel that if I had more time it would eventually break through, but it often stops too quickly.
